src/datasets/non_desktop/day_2_7_activation/reference.md

```diff
@@ -0,0 +1,48 @@
+# Non Desktop Day 2 - 7 Activation  
+
+<!-- toc -->
+
+# Introduction
+`firefox_nondesktop_day_2_7_activation` is designed for use in calculating the day 2-7 activation metric, a key result in 2020 for the nondesktop products.
+
+
+# Contents
+`firefox_nondesktop_day_2_7_activation` is an aggregate table with two key metrics: `new_profiles` and `day_2_7_activated`. These are primarily aggregated by `submission_date`.
+
+- `new_profiles`: Unique count of client ids with a given profile creation date. As not all initial pings are received exactly on the day of profile creation, we wait for 7 days after the profile creation date before establishing the New Profile cohort to ensure the data is complete.
+- `days_2_7_activated`: Unique count of client ids who use the product at any point starting the day after they created a profile up to 6 days after.
+
+Due to the delay in receiving all the initial pings and subsequent 7 day wait prior to establishing the new profile cohort, the table will lag other daily usage nondesktop tables e.g. `nondesktop_clients_last_seen` by 7 days. 
+
+We also include a variety of dimension information (e.g. `product`, `app_name`, `app_version`, `os`, `normalized_channel` and `country`) to aggregate on. 
+
+# Accessing the Data
+Access the data at [`moz-fx-data-shared-prod.telemetry.firefox_nondesktop_day_2_7_activation`](https://console.cloud.google.com/bigquery?project=moz-fx-data-shared-prod&p=moz-fx-data-shared-prod&d=telemetry&t=firefox_nondesktop_day_2_7_activation&page=table)
+
+# Data Reference
+## Example Queries
+[This activation query](https://sql.telemetry.mozilla.org/queries/72054/source) gives the `day 2-7 activation` by product. A new profile is considered activated if the client uses the browser at any point starting 1 day up to 6 days after the `profile creation date`.
```

Comment:
Let's put the SQL inline:

```suggestion

This query gives the `day 2-7 activation` by product:

```sql
SELECT
  cohort_date,
  product,
  SUM(day_2_7_activated) as day_2_7_activated,
  SUM(new_profiles) as new_profiles,
  SAFE_DIVIDE(SUM(day_2_7_activated), SUM(new_profiles)) as day_2_7_activation
FROM
  `moz-fx-data-derived-datasets.telemetry.firefox_nondesktop_day_2_7_activation`
WHERE
  cohort_date = "2020-03-01"
GROUP BY 1,2
ORDER BY 1
```
[link](https://sql.telemetry.mozilla.org/queries/72054/source)

A new profile is considered activated if the client uses the browser at any point starting 1 day up to 6 days after the `profile creation date`.
```